bin/check-mysql-replication-status.rb

```diff
@@ -93,6 +94,27 @@ class CheckMysqlReplicationStatus < Sensu::Plugin::Check::CLI
          # #YELLOW
          proc: lambda { |s| s.to_i } # rubocop:disable Lambda
 
+  option :flapping_lag,
+         short: '-l',
+         long: '--flapping-lag=VALUE',
+         description: 'Lag threshold to trigger flapping protection',
+         default: 100000,
+         proc: lambda { |s| s.to_i } # rubocop:disable Lambda
+
+  option :flapping_retry,
+         short: '-r',
+         long: '--flapping-retry=VALUE',
+         description: 'Number of retries when lag flapping protection is triggered',
+         default: 0,
+         proc: lambda { |s| s.to_i } # rubocop:disable Lambda
+
+  option :flapping_sleep,
+         long: '--flapping-sleep=VALUE',
+         description: 'Sleep between flapping protection retries',
+         default: 1,
+         proc: lambda { |s| s.to_i } # rubocop:disable Lambda
```

Comment:
Any particular reason to use a lambda over a proc that I am not seeing? This would remove the need to disable the cop.

```suggestion
         proc: proc { |s| s.to_i }
```

Reply:
Addressed in latest rework